src/lib/y2storage/autoinst_proposal.rb

```diff
@@ -296,6 +283,8 @@ def add_reduced_devices_issues(result)
     # @see Y2Storage::BlkDevice#name
     def proposal_settings_for_disks(drives)
       settings = @proposal_settings || ProposalSettings.new_for_current_product
+      # FIXME: #use_snapshots is a property of legacy settings format. To configure snapshots with the
```

Comment:
In this case, we need to 1) open a new bug report or 2) fix it now. It is a real problem, although I would say that this is not the most common use case for AutoYaST. We already have 35 TODOs and 83 FIXMEs :smile:. Not to mention the 4 DEPRECATED notes.

Reply:
After discussing about it, finally I have also fixed this issue.